test/functional/sqlsrv/MsHelper.inc

```diff
@@ -462,22 +456,38 @@ function selectFromTable($conn, $tbname, $conds = null, $values = null)
  * Perform a query from a table with or without where condition(s)
  * @param  resource  $conn : connection resource
  * @param  string  $sql : T-SQL query
- * @param  string  $conds : string of condition(s) with placeholders, null by default
+ * @param  string  $conds : string of condition(s) possibly with placeholders, null by default
  * @param  array  $values : array of parameters, null by default
- * @return object sqlsrv statement upon success or false otherwise
+ * @return resource sqlsrv statement upon success or false otherwise
  */
 function executeQuery($conn, $sql, $conds = null, $values = null)
 {
-    // when AE is enabled, use sqlsrv_prepare() to handle fields with unlimited size
-    if (!isColEncrypted() && (is_null($conds) || empty($values))) {
+    if (!isColEncrypted()) {
+        if (!empty($values)) {
```

Comment:
can you add a comment saying this if block is for replacing '?' placeholders with values